node-build.sh

```diff
@@ -5,8 +5,27 @@ set -e
 # enable nullglob - allows filename patterns which match no files to expand to a null string, rather than themselves
 shopt -s nullglob
 
+# parse flags
+while getopts ":vt" opt; do
+  case $opt in
+    v) VERBOSE=true;;
+    t) WITH_TESTING=true;;
+  esac
+done
+
 DIST="./dist"
 
+# show input environment args
+if [ $VERBOSE ]
+then
+    echo "env: BITBUCKET_CLONE_DIR=$BITBUCKET_CLONE_DIR"
+    echo "env: MYGET_ACCESS_TOKEN=$MYGET_ACCESS_TOKEN" | sed -e 's/=.\+/=******/g'
```

Comment:
Still not a fan of including a sensitive secret like this ;)

Reply:
Just for debugging